### PR TITLE
- fixes #8231 - aligns terminology to clearly differenciate between notifications and change notifications

### LINKED ISSUE #8231: Avoid the confusion between 'subscriptions to change notifications' and 'notifications'
please add a clarification the clearly distinguish between 

- *subscriptions* to change notification (applications)   `/subscriptions`
- human notification `/me/notifications/`

as the articles for 'change notification' often simply state 'notification' there is a risk for confusion with the newly introduced ['Microsoft Graph notifications' API](https://docs.microsoft.com/en-us/graph/notifications-concept-overview)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 33d14c58-7163-a7b7-6f84-c075d0f6870d
* Version Independent ID: 3710d019-8a67-59cc-75b6-bcecc106180d
* Content: [Use the Microsoft Graph API to get change notifications - Microsoft Graph v1.0](https://docs.microsoft.com/en-us/graph/api/resources/webhooks?view=graph-rest-1.0)
* Content Source: [api-reference/v1.0/resources/webhooks.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/v1.0/resources/webhooks.md)
* Product: ****
* Technology: **microsoft-graph**
* GitHub Login: @baywet
* Microsoft Alias: **MSGraphDocsVteam**

### PR DESCRIPTION
**Pull request by baywet**

### CODE REVIEW

File: api-reference/beta/resources/subscription.md [hunk 1]
```diff
@@ -40,17 +40,17 @@ See [Use the Microsoft Graph API to get change notifications](webhooks.md) for t
 
 | Property | Type | Description |
 |:---------|:-----|:------------|
-| changeType | string | Indicates the type of change in the subscribed resource that will raise a notification. The supported values are: `created`, `updated`, `deleted`. Multiple values can be combined using a comma-separated list. Required. <br><br>Note: Drive root item and list notifications support only the `updated` changeType. User and group notifications support `updated` and `deleted` changeType. |
-| notificationUrl | string | The URL of the endpoint that receives the notifications. This URL must make use of the HTTPS protocol. Required. |
+| changeType | string | Indicates the type of change in the subscribed resource that will raise a cahnge notification. The supported values are: `created`, `updated`, `deleted`. Multiple values can be combined using a comma-separated list. Required. <br><br>Note: Drive root item and list change notifications support only the `updated` changeType. User and group change notifications support `updated` and `deleted` changeType. |
```
Comment (davidmu1) [inline api-reference/beta/resources/subscription.md:43]: cahge should be change

Comment (baywet) [reply api-reference/beta/resources/subscription.md:43]:
```suggestion
| changeType | string | Indicates the type of change in the subscribed resource that will raise a change notification. The supported values are: `created`, `updated`, `deleted`. Multiple values can be combined using a comma-separated list. Required. <br><br>Note: Drive root item and list change notifications support only the `updated` changeType. User and group change notifications support `updated` and `deleted` changeType. |
```

File: concepts/webhooks-with-resource-data.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+Including resource data as part of change notifications requires you to implement the following additional logic to satisfy data access and security requirements: 
 
-- [Handle](#subscription-life-cycle-notifications) special subscription _life cycle_ notifications to maintain an uninterrupted flow of data. Microsoft Graph sends life cycle notifications from time to time to require an app to re-authorize, to make sure access issues have not unexpectedly cropped up for including resource data in notifications.
-- [Validate](#validating-the-authenticity-of-notifications) the authenticity of notifications as having originated from Microsoft Graph.
-- [Provide](#decrypting-resource-data-from-change-notifications) a public encryption key and use a private key to decrypt resource data received through notifications.
+- [Handle](#subscription-lifecycle-notifications) special subscription _lifecycle_ notifications to maintain an uninterrupted flow of data. Microsoft Graph sends lifecycle notifications from time to time to require an app to re-authorize, to make sure access issues have not unexpectedly cropped up for including resource data in change notifications.
```
Comment (davidmu1) [inline concepts/webhooks-with-resource-data.md:15]: I don't think the italics is necessary.

File: concepts/webhooks-with-resource-data.md [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
+- [Validate](#validating-the-authenticity-of-notifications) the authenticity of change notifications as having originated from Microsoft Graph.
+- [Provide](#decrypting-resource-data-from-change-notifications) a public encryption key and use a private key to decrypt resource data received through change notifications.
 
 ## Resource data in notification payload
 
-In general, this type of change notifications include the following resource data in the payload:
+In general, this type of change change notifications include the following resource data in the payload:
```
Comment (davidmu1) [inline concepts/webhooks-with-resource-data.md:21]: The additional change should be removed.

File: concepts/webhooks-with-resource-data.md (hunk 2 above)
Comment (baywet) [reply concepts/webhooks-with-resource-data.md:15]:
```suggestion
- [Handle](#subscription-lifecycle-notifications) special subscription lifecycle notifications to maintain an uninterrupted flow of data. Microsoft Graph sends lifecycle notifications from time to time to require an app to re-authorize, to make sure access issues have not unexpectedly cropped up for including resource data in change notifications.
```

File: concepts/webhooks-with-resource-data.md (hunk 3 above)
Comment (baywet) [reply concepts/webhooks-with-resource-data.md:21]:
```suggestion
In general, this type of change notifications include the following resource data in the payload:
```

File: concepts/webhooks-with-resource-data.md [hunk 4]
```diff
@@ -91,26 +91,26 @@ Content-Type: application/json
 }
 ```
 
-## Subscription life cycle notifications
+## Subscription lifecycle notifications
 
-Certain events can interfere with normal notification flow in an existing subscription. Subscription _life cycle notifications_ inform you actions to take in order to maintain an uninterrupted flow. Unlike a resource change notification which informs a change to a resource instance, a life cycle notification is about the subscription itself, and its current state in the life cycle. 
+Certain events can interfere with change notification flow in an existing subscription. Subscription _lifecycle notifications_ inform you actions to take in order to maintain an uninterrupted flow. Unlike a resource change notification which informs a change to a resource instance, a lifecycle notification is about the subscription itself, and its current state in the lifecycle.
```
Comment (davidmu1) [inline concepts/webhooks-with-resource-data.md:96]: I don't think the italics is necessary here.

Comment (baywet) [reply concepts/webhooks-with-resource-data.md:96]:
```suggestion
Certain events can interfere with change notification flow in an existing subscription. Subscription lifecycle notifications inform you actions to take in order to maintain an uninterrupted flow. Unlike a resource change notification which informs a change to a resource instance, a lifecycle notification is about the subscription itself, and its current state in the lifecycle. 
```

**Pull request merged**